src/wizards/tokenSwapDealWizard/components/tokenDetails/tokenDetails.html

```diff
@@ -0,0 +1,158 @@
+<template>
+  <pcard if.bind="viewMode === 'edit'">
+    <pform-input
+      helper-message.bind="tokenInfoLoading ? 'Searching token address details...': ''"
+      input-reference.bind="addressInput"
+      label="Token Address"
+      label-info="Some info here"
+    >
+      <pinput-text
+        input.trigger="getTokenInfo(token.address)"
+        placeholder="0x..."
+        validation-state.bind="tokenInfoLoading? 'validating': addressInput.validationState !== 'validating' ? addressInput.validationState : undefined"
+        value.bind="token.address & validate"
+        view-model.ref="addressInput"
+      ></pinput-text>
+    </pform-input>
+    <div if.bind="infoSearched">
+      <table class="detailsTable">
+        <tr>
+          <td>
+            <p>Name</p>
+            <p if.bind="token.name">${token.name}</p>
+            <pinput-text if.bind="!token.name" value.bind="token.name"></pinput-text>
+          </td>
+          <td>
+            <p>Symbol</p>
+            <p if.bind="token.symbol">${token.symbol}</p>
+            <pinput-text if.bind="!token.symbol" value.bind="token.symbol"></pinput-text>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <p>Decimals</p>
+            <p if.bind="token.decimals">${token.decimals}</p>
+            <pinput-numeric if.bind="!token.decimals" value.bind="token.decimals"></pinput-numeric>
+          </td>
+          <td>
+            <p>Logo <img height="24" src.bind="token.logoURI"></p>
+            <pinput-text value.bind="token.logoURI"></pinput-text>
+          </td>
+        </tr>
+      </table>
```

Comment:
Just random related thought:
Wondering what is easier to maintain `<table>` or `display: grid`

Reply:
I much prefer using div+grid.  I very rarely resort to using table.  Grid makes is more clear what behavior is required, and fewer permutations of technologies we have to work with, particularly in mobile responsiveness situations.